sys/dev/acpica/acpi_apei.c

```diff
@@ -779,5 +811,23 @@ static driver_t	apei_driver = {
 	sizeof(struct apei_softc),
 };
 
-DRIVER_MODULE(apei, acpi, apei_driver, apei_devclass, 0, 0);
+static int
+apei_modevent(struct module *mod __unused, int evt, void *cookie __unused)
+{
+	int err = 0;
+
+	switch (evt) {
+	case MOD_LOAD:
+		sysctl_ctx_init(&apei_sysctl_ctx);
+		break;
+	case MOD_UNLOAD:
+		sysctl_ctx_free(&apei_sysctl_ctx);
+		break;
+	default:
+		err = EINVAL;
```

Comment:
`module(9)` says unsupported/unrecognized values of `what` should return `EOPNOTSUPP`, but I see quite a few modules returning `EINVAL`, so I guess either works.